tools/test/test_codegen.py

```diff
@@ -217,6 +217,11 @@ def setUp(self) -> None:
             loc=torchgen.model.Location(__file__, 1),
             valid_tags=set(),
         )
+        self.fragment_custom_native_function, _ = torchgen.model.NativeFunction.from_yaml(
+            {"func": "custom::func() -> bool"},
+            loc=torchgen.model.Location(__file__, 1),
+            valid_tags=set("fragment"),
+        )
```

Comment:
Other than that can you add a test to parse `tags: fragment` from yaml file?

Reply:
OK, sounds good